docs/TRAINING_STATISTICS.md

```diff
@@ -3,13 +3,15 @@
 Godot RL Agents uses [Tensorboard](https://www.tensorflow.org/tensorboard) to log training statistics. You can start Tensorboard by running the following command:
 
 ```bash
-tensorboard --logdir ./logs/[RL_FRAMEWORK] -p 7000
```

Comment:
I removed this because it causes an error: `tensorboard: error: unrecognized arguments: -p 7000`. Down below I changed the port to the default 6006.